Form-Controls/index.html

```diff
@@ -14,15 +14,80 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
+    <div>
+      <label>Name</label>
+      <input name="Name" id="name" required>
+    </div>
+    <div>
+      <label>Email</label>
+      <input type="email" name="Email" id="Email" required>
+    </div>
+    
+    <div>
+      <label>T-Shirt Colour</label>
+      <select>
+        <option>
+          Blue
+      
+        </option>
+        <option>
+          Red
+        </option>
+        <option>
+          Yellow
+        </option>
+      </select>
+    </div>
+
+    <div>
+      <label>T-Shirt Size</label>
+      <select>
+        <option>
+          XS
+      
+        </option>
+        <option>
+          S
+      
+        </option>
+        <option>
+          M
+        </option>
+        <option>
+          L
+        </option>
+        <option>
+          XL
+      
+        </option>
+        <option>
+          XXL
+      
+        </option>
+      </select>
```

Comment:
Same comment as the above Select element, add an id the select and the for attribute to the label.